pythran/tables.py

```diff
@@ -873,6 +873,12 @@ def update_effects(self, node):
     logger.warn("Pythran support disabled for module: omp")
     del MODULES["omp"]
 
+# copyto and isclose are not always supported by numpy
+unsupported_numpy_methods = ('copyto', 'isclose')
```

Comment:
It's wrong because it fails later on during argument introspection, and
probably constant folding too.

So basically, if you have a slightly old numpy version, pythran crahses
all the time. This fixes that.